### PR TITLE
Improve the API documentation and markdown files

### PR DESCRIPTION
**Pull request by ldclakmal**
## Purpose
$subject
Related to: https://github.com/ballerina-platform/ballerina-standard-library/issues/584

## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: url-ballerina/Package.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
 
-This package provides the necessary utilities, which are required to encode and decode a URL or part of a URL.
+The URL library is one of the standard library modules of the [Ballerina](https://ballerina.io/) language.
 
-For information on the operations, which you can perform with this package, see the below **Functions**. For an example on the usage of the operations, see the [URL Encode/Decode Operations](https://ballerina.io/learn/by-example/url-encode-decode.html).
+URL encoding stands for encoding certain characters in a URL by replacing them with one or more character triplets that consist of the percent character "%" followed by two hexadecimal digits. The two hexadecimal digits of the triplet(s) represent the numeric value of the replaced character.
+
+The Ballerina URL module facilitates APIs to encode and decode a URL or part of a URL.
+
+For more information go to [The URL Module](https://docs.central.ballerina.io/ballerina/url/latest).
```
Comment (daneshk) [inline url-ballerina/Package.md:9]: shall we remove API Doc link from package.md. In central, there is a link to API Docs.

Comment (ldclakmal) [reply url-ballerina/Package.md]: Fixed with https://github.com/ballerina-platform/module-ballerina-url/pull/157/commits/bd4ac6a47962b94e53fc09a57ea157801831a4b4

File: README.md [hunk 2]
```diff
@@ -88,6 +90,6 @@ All contributors are encouraged to read the [Ballerina Code of Conduct](https://
 
 ## Useful Links
 
-* Discuss about code changes of the Ballerina project in [ballerina-dev@googlegroups.com](mailto:ballerina-dev@googlegroups.com).
+* Discuss the code changes of the Ballerina project in [ballerina-dev@googlegroups.com](mailto:ballerina-dev@googlegroups.com).
```
Comment (MadhukaHarith92) [inline README.md:93]:
```suggestion
* Discuss code changes of the Ballerina project in [ballerina-dev@googlegroups.com](mailto:ballerina-dev@googlegroups.com).
```

File: url-ballerina/Package.md [hunk 3]
```diff
@@ -1,5 +1,17 @@
...
+
+The Ballerina URL module facilitates APIs to encode and decode a URL or part of a URL.
+
+### Report Issues
+
+To report bugs, request new features, start new discussions, view project boards, etc. please visit Ballerina
+[Standard Library parent repository](https://github.com/ballerina-platform/ballerina-standard-library).
+
+### Useful Links
+- Discuss the code changes of the Ballerina project in [ballerina-dev@googlegroups.com](mailto:ballerina-dev@googlegroups.com).
```
Comment (MadhukaHarith92) [inline url-ballerina/Package.md:15]:
```suggestion
- Discuss code changes of the Ballerina project in [ballerina-dev@googlegroups.com](mailto:ballerina-dev@googlegroups.com).
```

File: README.md [hunk 4]
```diff
@@ -3,21 +3,23 @@ Ballerina URL Library
...
-The URL library is one of the standard library modules of the<a target="_blank" href="https://ballerina.io/"> Ballerina</a> language.
+The URL library is one of the standard library modules of the [Ballerina](https://ballerina.io/) language.
 
-It provides the necessary utilities, which are required to encode and decode a URL or part of a URL.
+URL encoding stands for encoding certain characters in a URL by replacing them with one or more character triplets that consist of the percent character "%" followed by two hexadecimal digits. The two hexadecimal digits of the triplet(s) represent the numeric value of the replaced character.
 
-For more information go to [The URL Module](https://ballerina.io/learn/api-docs/ballerina/url/index.html).
+The Ballerina URL module facilitates APIs to encode and decode a URL or part of a URL.
+
+For more information go to [The URL Module](https://docs.central.ballerina.io/ballerina/url/latest).
```
Comment (praneesha) [inline README.md:16]:
```suggestion
For more information, go to the [URL module](https://docs.central.ballerina.io/ballerina/url/latest).
```

File: README.md [hunk 5]
```diff
@@ -32,7 +34,14 @@ This repository only contains the source code for the module.
    * [OpenJDK](https://adoptopenjdk.net)
    
         > **Note:** Set the JAVA_HOME environment variable to the path name of the directory into which you installed JDK.
-     
+
+2. Export GitHub Personal Access Token (PAT) with 'read package' permission as follows:
```
Comment (praneesha) [inline README.md:38]:
```suggestion
2. Export your GitHub Personal Access Token (PAT) with the 'read package' permission as follows:
```

**Review by MadhukaHarith92 — APPROVED** [review]
LGTM

**Review by praneesha — APPROVED** [review]

**Pull request merged**